automatic/alldup/tools/chocolateyInstall.ps1

```diff
@@ -1,6 +1,19 @@
-﻿$packageName = '{{PackageName}}'
-$installerType = 'exe'
-$silentArgs = '/VERYSILENT'
-$url = '{{DownloadUrl}}'
+﻿$packageName  = 'alldup'
+$url          = 'http://www.alldup.info/download/alldup.exe'
+$checksum     = '18f7b97880f6b56c9dfa621d6b6c0ec04edbb1b7923c93bffd72952effdc91e2'
+$checksumType = 'sha256'
 
-Install-ChocolateyPackage $packageName $installerType $silentArgs $url
+$packageArgs  = @{
+  packageName    = $packageName
+  fileType       = 'exe'
+  softwareName   = 'AllDup*'

```

Comment:
I honestly don't know.
I just added it because of:
1. It is part of the template when running `choco new`
2. I thought perhaps it had something to do with Synchronizing with Porgams and features when using the Pro version.

I can remove it though if it truly has no meaning.